src/index.ts

```diff
@@ -1 +1,3 @@
 export * from "./commands";
+export * from "./settings";
+export * from "./logger";
```

Comment:
(Add any other named exports you want to expose here)

```suggestion
export { Logger } from "./logger";
```